drivers/hmis/app/graphql/types/hmis_schema/has_assessments.rb

```diff
@@ -30,6 +30,10 @@ def resolve_assessments(scope = object.assessments, **args)
         apply_assessment_arguments(scope, **args)
       end
 
+      def resolve_assessments_including_wip(scope = object.assessments_including_wip, **args)
+        apply_assessment_arguments(scope, **args)
+      end
+
```

Comment:
@ZachCMP should I be using the data loader (load_ar_association) here? I don't know if we want/need it here. This is only used for loading assessments associated with 1 enrollment.

Reply:
@gigxz Yeah, if this is only currently used in the context of one enrollment, then it's probably fine to omit it. If we're in a case where we want to fetch assessments for multiple enrollments in the same query, then we probably want to look at adding it to prevent N+1.

Reply:
Ok, got it!